src/Core/Mesh/TopologicalTriMesh/Operations/EdgeSplit.cpp

```diff
@@ -31,20 +31,20 @@ void splitEdge( TopologicalMesh& topologicalMesh, TopologicalMesh::EdgeHandle ed
 
     CORE_ASSERT( fraction > 0 && fraction < 1, "Invalid fraction" );
 
-    TopologicalMesh::VertexHandle v1 =
-        topologicalMesh.to_vertex_handle( topologicalMesh.halfedge_handle( edgeHandle, 0 ) );
-    TopologicalMesh::VertexHandle v2 =
-        topologicalMesh.to_vertex_handle( topologicalMesh.halfedge_handle( edgeHandle, 1 ) );
+    TopologicalMesh::HalfedgeHandle he0 = topologicalMesh.halfedge_handle( edgeHandle, 0 );
+    TopologicalMesh::HalfedgeHandle he1 = topologicalMesh.halfedge_handle( edgeHandle, 1 );
+    TopologicalMesh::VertexHandle v1 = topologicalMesh.to_vertex_handle( he0 );
+    TopologicalMesh::VertexHandle v2 = topologicalMesh.to_vertex_handle( he1 );
 
     TopologicalMesh::Point p = TopologicalMesh::Point(
         fraction * topologicalMesh.point( v1 ) + ( 1. - fraction ) * topologicalMesh.point( v2 ) );
-    TopologicalMesh::Normal n =
-        TopologicalMesh::Normal( ( fraction * topologicalMesh.normal( v1 ) +
-                                   ( 1. - fraction ) * topologicalMesh.normal( v2 ) )
-                                     .normalized() );
+
+    //        TopologicalMesh::Normal n =  TopologicalMesh::Normal((fraction *
+    //        topologicalMesh.normal(v1) + (1. - fraction) *
+    //        topologicalMesh.normal(v2)).normalized());
 
     TopologicalMesh::VertexHandle vh = topologicalMesh.add_vertex( p );
-    topologicalMesh.set_normal( vh, n );
+    //        topologicalMesh.set_normal(vh,n);
     topologicalMesh.split( edgeHandle, vh );
```

Comment:
What's the expected behavior of the split operation regarding vertex attribute ?
Is the normal vector still handled ?

Reply:
not handle for now ... need further dev (but time is missing ...)

Reply:
What is the interest of adding user defined attributes if they are not handled thoroughly by all operators ?
Apart time, what are the difficulties of developing such attribute management ?

Reply:
Well... time. 
We believe this could be added in an upcoming PR.